src/xdocs/config_javadoc.xml

```diff
@@ -3619,7 +3619,7 @@ public int foobar() {
           Checks that
           <a href="https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#firstsentence">
           Javadoc summary sentence</a> does not contain phrases that are not recommended to use.
-          Summaries that contain only the <code>{@inheritDoc}</code> tag are skipped. Check also
+          Summaries that contain only the {@code {@inheritDoc}} tag are skipped. Check also
```

Comment:
To me, the previous version looks correct since the new code isn't html.

Reply:
Done.